nbcelltests/tests/test_test.py

```diff
@@ -637,6 +638,40 @@ def test_input_celltest_injection_comment(self):
         self.t._run("assert x == 1")
 
 
+class TestKernelCwd(_TestCellTests):
+    """Do we remember to set cwd?"""
+
+    NBNAME = KERNEL_CWD
```

Comment:
This doesn't seem very intuitive. Is this something that users are expected to use, or is this only for our test?

Reply:
Oh, aha. I see what tripped me: This is not a directory path, but the path to a notebook testing CWDs...

Reply:
Yes, confusing! There are entirely too many classes with Test in the name in this project :) What will improve this is generating notebooks as needed in place rather than having various ones stored on disk. There's an issue to do this cleanup already.